app/views/issues/_table.html.erb

```diff
@@ -31,7 +31,7 @@
                   [issue.title, (link_to(issue.title, [current_project, issue], data: { qa_visible: false } ) + link_to(issue.title, project_qa_issue_path(current_project, issue), class: 'd-none', data: { qa_visible: true }))]
                 when 'Tags'
                   [
-                    issue.tags.any? ? issue.tags.first.display_name : '',
+                    issue.tags.any? ? issue.tags.first.position : false,
```

Comment:
We are using ints to sort tags: 0 = highest priority, next comes 1, then 2, and so on. 

Due to how JS sorts, having a `''` or 'null' value for `sort` will keep untagged issues above the highest priority issue (ie "Critical"), since `''`/`null` will come before 0. Strings/bools sort after the ints so having a bool or a string, allows the untagged issues to be sorted after the issues tagged with the lowest priority tag. I went with `false` as it seemed the most reasonable over an arbitraray string.